src/matlab2tikz.m

```diff
@@ -3514,14 +3514,7 @@ function warnFacetedInterp(m2t)
         cData = get(h, 'MarkerEdgeColor');
     end
 
-    matlabMarker = get(h, 'Marker');
-    markerFaceColor = get(h, 'MarkerFaceColor');
-    markerEdgeColor = get(h, 'MarkerEdgeColor');
-    hasFaceColor = ~isNone(markerFaceColor);
-    hasEdgeColor = ~isNone(markerEdgeColor);
-    markOptions = opts_new();
-    [tikzMarker, markOptions] = translateMarker(m2t, matlabMarker, ...
-        markOptions, hasFaceColor);
+    rawMarker = getMarkerStruct(m2t, h);
```

Comment:
I fully agree that `rawMarker` is not optimal. 
However, we typically use `*Options` and `*Opts` for data that is structured in a particular way rather than such a struct, so I think those are not any better.